solidity/contracts/bridge/Bridge.sol

```diff
@@ -771,6 +742,53 @@ contract Bridge is Ownable, EcdsaWalletOwner {
         // TODO: Handle deposits having `vault` set.
     }
 
+    /// @notice Resolves sweeping wallet based on the provided wallet public key
+    ///         hash. Validates the wallet state and current main UTXO, as
+    ///         currently known on the Ethereum chain.
+    /// @param walletPubKeyHash public key hash of the wallet proving the sweep
+    ///        Bitcoin transaction.
+    /// @param mainUtxo Data of the wallet's main UTXO, as currently known on
+    ///        the Ethereum chain. If no main UTXO exists for the given wallet,
+    ///        this parameter is ignored
+    function resolveWallet(
```

Comment:
That function has some requirements, can we list them in `@dev` section?
```
/// @dev Requirements:
         - Sweeping wallet must be either in Live or MovingFunds state
         - If the main UTXO of the sweeping wallet exists in the storage, the passed `mainUTXO`
           parameter must be equal to the stored one
```

Reply:
Good idea. 3fc6675b07f30a32c35de58dabeee9a426ecb92c